src/main/java/uk/gov/hmcts/reform/orgrolemapping/config/DBFlagConfigurtion.java

```diff
@@ -0,0 +1,59 @@
+package uk.gov.hmcts.reform.orgrolemapping.config;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.orgrolemapping.data.FlagConfig;
+import uk.gov.hmcts.reform.orgrolemapping.data.FlagConfigRepository;
+import uk.gov.hmcts.reform.orgrolemapping.domain.model.enums.FeatureFlagEnum;
+
+import java.util.concurrent.ConcurrentHashMap;
+
+@Component
+@Slf4j
+public class DBFlagConfigurtion implements CommandLineRunner {
```

Comment:
Please use @Configuration instead of @Component as it a configuration file.

Reply:
Done